package.json

```diff
@@ -34,7 +34,7 @@
     "site:clean": "rimraf './_site/!(assets)' './_site/assets/!(images)'",
     "assets:clean": "rimraf './_built/'",
     "serve": "npm-run-all build:assets build:site -p eleventy:serve watch:js watch:sass",
-    "test": "jest",
+    "test": "jest -i",
```

Comment:
There seems to be a bug with using multiple workers in Jest 29, so this limits it to one worker. I'll file an issue.

Reply:
https://github.com/facebook/jest/issues/13332